docs/installation-instructions.rst

```diff
@@ -0,0 +1,74 @@
+Installation Instructions
+===========================
+
+The following step by step guide will get you started with XMC for Arduino:
+
+Prework for SEGGER J-Link
+------------------------------
+
+In order to use Infineon's XMC microcontroller boards in Arduino you need
+`SEGGER J-Link <https://www.segger.com/downloads/jlink>`_ installed on your machine first. Please follow
+`this link <https://www.segger.com/downloads/jlink>`_ 
+and install the J-Link Software and Documentation Pack for your respective operating system (OS).
+
+.. image:: img/jlink_install.png
+    :width: 700
+
+If you are downloading J-Link for the first time, make sure to check this box: Install USB Driver for J-Link.
+
+.. image:: img/J-Link_Installer_options_page.png
+    :width: 650
+
+Required tools
```

Comment:
Sorry I updated the content here, it needs to be added: https://github.com/Infineon/XMC-for-Arduino?tab=readme-ov-file#required-tools
pip's link was displayed incorrectly

Reply:
@CaGriesser